tools/correlation_analysis/abims_correlation_analysis.xml

```diff
@@ -0,0 +1,322 @@
+<tool id="correlation_analysis" name="Metabolites Correlation Analysis" version="1.0.1+galaxy0" >
+
+    <description></description>
+
+    <requirements>
+        <requirement type="package" version="1.1_5">r-batch</requirement>
+        <requirement type="package" version="0.8.8">r-reshape</requirement>
+        <requirement type="package" version="7.3_53">r-mass</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" level="fatal" />
```

Comment:
this can go aways with the new `detect_errors` on the command tag: https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection

Reply:
Done